src/browser.js

```diff
@@ -51,13 +51,19 @@ const plugin =
           });
         }
         log(level, ...args) {
-          return emitter.emit('universal-log', {
+          emitter.emit('universal-log', {
             level,
             args: args.map(normalizeErrors),
           });
+
+          // send errors immediately instead of batching to prevent
+          // unwieldy batch sizes
+          if (level === 'error') {
+            // $FlowFixMe
+            emitter.flush().catch(error => this.log('error', error));
```

Comment:
Flushing here might change the frequency of unrelated events. Probably not a huge dealing but just calling it out here in case there are unintended consequences

Reply:
Yeah, this isn't as clean cut as it could be, but the effort to send a truly isolated batch for each error outweighs the benefit it would provide over this approach imo. If it somehow becomes a problem for someone in the future, we can do it more properly

Reply:
regarding `.catch()`- is the intention here to re-send the error? `fusion-plugin-universal-events` already handles fail-safe except for getting batched items from the storage with `this.storage.getAndClear()` https://github.com/fusionjs/fusion-plugin-universal-events/blob/b4c7b85ed71e4c7ae4745185232df2f585702c63/src/browser.js#L53-L76
otherwise `.flush()` won't reject

Reply:
Was just being overly cautious. You're right that universal events eats pretty much any error that could happen. I can remove it if you think it's redundant.

The only case I could see it being helpful in is if, at some point in the future, universal-events gets updated and no longer handles its own errors

Reply:
actually please disregard the prev comment, I misread the variables and now realize you are handling exactly just the rejection case from the emitter. Another concern here is that the batch might bloat quicking if `.flush()` is failing consistently for a period of time